server/prisma/schema.prisma

```diff
@@ -26,9 +26,18 @@ model User {
   createdAt                DateTime                   @default(now())
   googleId                 String?                    @unique
   facebookId               String?                    @unique
+  Token                    Token[]
   UsersHasActionsReactions UsersHasActionsReactions[]
 }
 
+model Token {
```

Comment:
Why did you create a Token model? What is its purpose?

Reply:
The Goal of the Token model is to store all the AccessToken of a User.
It's useful to use the Service API as the User :)